CAFMaker/CAFMaker_module.cc

```diff
@@ -141,6 +141,7 @@ namespace caf {
       fCAFFilename.resize(dotpos);
       fCAFFilename += fParams.FileExtension();
 
+      std::cout << fCAFFilename << std::endl;
```

Comment:
prefer to make use of the `messagefacility` : 
mf::LogInfo("CAFMaker") << "Output CAF filename is " << fCAFFilename;